index.js

```diff
@@ -46,10 +49,10 @@ var gh = (function() {
         chrome.storage.sync.get({
           access_token: null,
         }, function(items) {
-          access_token = items.access_token;
+          access_token = items.access_token; 
           options = {
             'interactive': interactive,
-            url:'https://api.voxity.fr/api/v1/dialog/authorize?client_id=' + clientID +
+            url: base_url+'/api/v1/dialog/authorize?client_id=' + clientID +
```

Comment:
Is there any reason to add spacing or removing extra space ? Sorry for asking I just want learn from my faults